### PR TITLE
fix merge conflict >>>>>

### PR DESCRIPTION
**Pull request by Rick-Anderson**

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
@danroth27  review please

File: aspnet/fundamentals/configuration.rst [hunk 1]
```diff
@@ -83,13 +83,9 @@ When specifying files as configuration sources, you can optionally specify wheth
 One way to leverage the order precedence of ``Configuration`` is to specify default values, which can be overridden. In the console application below, a default value for the ``username`` setting is specified in a ``MemoryConfigurationSource``, but this is overridden if a command line argument for ``username`` is passed to the application. You can see in the output how many different configuration sources are configured in the application at each stage of its execution.
 
 .. literalinclude:: configuration/sample/src/ConfigConsole/Program.cs
-<<<<<<< HEAD
-  :language: c#
   :emphasize-lines: 22,25
-=======
   :linenos:
```
Comment (danroth27) [inline aspnet/fundamentals/configuration.rst]: Remove linenos

**Comment by danroth27** [comment]
:shipit:

**Comment by Rick-Anderson** [comment]
@danroth27 I'll wait for Travis to complete before I merge. Failures show up in the first couple of minutes so I'm sure it will pass.

**Pull request closed without merging**